lib/utils/testLauncherHelper.js

```diff
@@ -104,9 +104,14 @@ function followChildProcess(child, callback) {
  */
 function validateInput(type, args) {
 	try {
+		let msg = texts['errorType.suitestCommand'](type.toLowerCase());
```

Comment:
I would collapse the definition of `msg` with the ternary operator so that it is easier to understand the flow.